hat/assets/js/apps/Iaso/domains/instances/components/InstancesFilesListComponent.tsx

```diff
@@ -105,6 +117,18 @@ const InstancesFilesList: FunctionComponent<Props> = ({
     if (files.length === 0) {
         return <Box p={2}>{formatMessage(MESSAGES.missingFile)}</Box>;
     }
+
+    const FormSubmissionLink = ({ images, imageIndex }) => {
+        const currentFormSubmissionUrl = `/forms/submission/instanceId/${images[imageIndex].itemId}`;
+        return (
+            <Box className={classes.link}>
+                <Link to={currentFormSubmissionUrl}>
+                    {formatMessage(MESSAGES.formSubmissionLinkLabel)}
+                </Link>
+            </Box>
+        );
+    };
+
```

Comment:
Please move that component out of the body of `InstanceFilesList` (along with its styling)